cp3pt0-deployment/uninstall_tenant.sh

```diff
@@ -69,6 +74,10 @@ function parse_arguments() {
             shift
             OPERATOR_NS=$1
             ;;
+        --retain-ns)
+            shift
+            RETAIN="true"
+            ;;
         -f)
             shift
             FORCE_DELETE=1
```

Comment:
```suggestion
        --retain-ns)
            RETAIN="true"
            ;;
        -f)
            FORCE_DELETE=1
```

If I understand it right, there is no additional argument following either the flag `--retain-ns` or `-f`, we might not need `shift` in this case.

Reply:
So two things about the -f flag:
1. I want to change the function of the script as little as possible including possible parameters. Its possible this script is being used with set parameters already somewhere and I don't want to disrupt that. I want --retain-ns to be a separate "opt in" flag 
2. techincally, its role is to enable force delete of resources if the namespace gets stuck in terminating. Essentially, without this flag enabled, we do not run the remove_all_finalizers function which deletes all the resources the namespace is waiting on